ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/subscription/blockheaders/NewBlockHeadersSubscriptionServiceTest.java

```diff
@@ -84,7 +85,7 @@ public void shouldSendMessageWhenBlockAdded() {
 
     simulateAddingBlock();
 
-    verify(subscriptionManager)
+    verify(subscriptionManager, timeout(100))
```

Comment:
We really want to avoid using multiple threads in tests - it' makes everything much more complex.  If the `ExecutorService` was passed in (or if vertx is being used, have an interface that abstracts it) then we could use a stub that executes on the same thread rather than deferring to another thread and avoid multiple threads in tests.

If for some reason we do get stuck with multiple threads 100ms is way too short a timeout.  Needs to be more like 10 seconds.